hack/util.sh

```diff
@@ -443,14 +448,14 @@ reset_tmp_dir() {
 	local sudo="${USE_SUDO:+sudo}"
 
 	set +e
-	${sudo} rm -rf ${BASETMPDIR} &>/dev/null
+	${sudo} rm -rf "${BASETMPDIR}" &>/dev/null
 	if [[ $? != 0 ]]; then
 		echo "[INFO] Unmounting previously used volumes ..."
-		findmnt -lo TARGET | grep ${BASETMPDIR} | xargs -r ${sudo} umount
-		${sudo} rm -rf ${BASETMPDIR}
+		findmnt -lo TARGET | grep "${BASETMPDIR}" | xargs -r "${sudo}" umount
```

Comment:
That change might actually make this method better -- currently it's written as if someone were trying to remove the tmpdirs themselves, and on noticing that the `rm -rf` failed, they `umount` and try again - with the `for` loop we wouldn't need the `if`, we could just `umount` any that we find and remove afterword.